src/get-cache.ts

```diff
@@ -5,7 +5,7 @@ export function getCache() {
   return new LRU<number, string>({
     // cache max. 15000 tokens, that will use less than 10mb memory
     max: 15000,
-    // Cache for 1 minute less than GitHub expiry
+    // Cache for 59 minutes less than GitHub expiry
```

Comment:
Thanks for the PR, but it says 

> 1 minute _less than GitHub expiry_

GitHub documents the installation access token expiration time as 60 minutes. Maybe we can make the comment more clear to prevent misunderstandings? I don’t mind making it longer

Reply:
Oh, it was me misunderstanding that comment at the first glance.
Sorry about that.

Yes, I agree that it would be more clear to mention the actual expiration time clearly in the comment.
Thanks for reviewing, I can close this PR.

Reply:
How about changing it to

```js
// Cache for 59 minutes (1 minute less than GitHub expiry)
```

Reply:
That would be awesome! 👍 
I will commit that change now.

Reply:
Force pushed the change 😉